### ISSUE TITLE
Add .no-scroll (or similar) class to body when modal is open to prevent body from scrolling

### ISSUE THREAD

**Issue by sushant-j** [egoist/vue-slim-modal#2]
The use case I am working with has a full screen modal, which has content overflowing the container. In this case, when the user scrolls, even the original body is scrolling. This is unwanted behaviour.

Would it be possible for to add either a .no-scroll or similar class to body to prevent scrolling?
(Assuming that class would have overflow-y: hidden).

Thanks!

**Comment by egoist** [comment]
yeah this makes sense, maybe add a prop: `isBodyScrollable` while you can already achieve this manually

**Comment by sushant-j** [comment]
Would you be able to add the said prop, `isBodyScrollable`?

**Comment by egoist** [comment]
Can you make a pull request? if not I will do it later.

**Comment by sushant-j** [comment]
I am not exactly sure how to proceed, so it would be better if you could do it.
